src/components/TaxPayer/PersonFields.tsx

```diff
@@ -15,37 +16,41 @@ interface PersonFieldsProps<T extends Person> extends BaseFormProps {
   defaults?: T
   children?: ReactNode
   errors: DeepMap<Partial<Person>, FieldError>
+  person?: Person | null | undefined
```

Comment:
This should be able to be simply set as `person?: Person` since the question mark denotes that the person interface property is possibly undefined.  From looking at the code it doesn't look like person needs to be able to be set to null, only undefined.